zip-0311.rst

```diff
@@ -2,7 +2,436 @@
 
   ZIP: 311
   Title: Sapling Payment Disclosure
-  Owners: Deirdre Connolly <deirdre@zfnd.org>
-  Status: Reserved
+  Owners: Jack Grigg <jack@electriccoin.co>
+  Status: Draft
   Category: Standards / RPC / Wallet
   Discussions-To: <https://github.com/zcash/zips/issues/387>
+
+
+Abstract
+========
+
+This ZIP describes a mechanism and format for disclosing information about Sapling spends
+and outputs within a transaction. In the typical case, this means enabling a sender to
+present a proof that they transferred funds to a recipient's Sapling address.
+
+
+Motivation
+==========
+
+There are various situations where a proof-of-payment may be desired. For example:
+
+- A sender may need to prove that their payment was sent to a recipient, and available to
+  be received (following the Zcash protocol in-band).
+- A third party may need to verify that a payment between a given sender and recipient was
+  executed successfully.
+
+When a transaction involves only transparent addresses, proof-of-payment is simple: The
+sender provides the transaction ID, and the recipent examines the blockchain to confirm
+that the transaction was mined. A third party can also perform this verification if they
+know the transparent addresses of the involved parties.
+
+However, if the transaction involves shielded addresses, the blockchain by itself does not
+contain enough information to allow a record of the payment to be reconstructed and
+verified:
+
+=========== =========== ====== ========= ====== ==================
+Source      Destination Sender Recipient Amount Payment dislosure?
+=========== =========== ====== ========= ====== ==================
+Transparent Transparent 🔍      🔍        🔍    Not required
+Transparent Shielded    🔍      🔐        🔐    Required
```

Comment:
The amount isn't fully hidden: you're revealing the total amount of utxos used as input.